groovy/libs.groovy/nbproject/project.xml

```diff
@@ -44,6 +44,7 @@
                 <package>org.codehaus.groovy.control</package>
                 <package>org.codehaus.groovy.control.customizers</package>
                 <package>org.codehaus.groovy.control.customizers.builder</package>
+                <package>org.codehaus.groovy.control.io</package>
```

Comment:
This change should be associated with some versioning. Increment spec version of the `libs.groovy` module; make sure those who depend on the `org.codehaus.groovy.control.io` package require the new version.

Reply:
Removed. Already part of https://github.com/apache/netbeans/pull/3027